src/modules.ts

```diff
@@ -346,6 +346,43 @@ export function getKeywords(thePackage: NpmPackageData): string[] {
   return keywords
 }
 
+export async function importOrRequire(moduleDir: string) {
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-require-imports
+    const mod = require(moduleDir)
+
+    // Node 20.19.0 will load ESM modules with require
+    // https://nodejs.org/en/blog/release/v20.19.0
```

Comment:
How about _Starting with version 20.19.0 and 22 Node will load ESM modules with require_ ?